proposals/wac-ucr/index.bs

```diff
@@ -802,6 +802,29 @@ For example, if the data Carol and Oscar saw in the resume was
 background, she wouldn't want them to know that they were only seeing
 a filtered view.
 
+### Minimal Credential Disclosure ### {#uc-minimalcredentials1}
+
+To continue with the  [[#uc-whopermitted]] example, Oscar now wants to
+view Alice's `resume` which is not publically accessible. To gain access,
+he needs to authenticate with the right credentials. Oscar does not want
+to try out each one of his credentials one by one until he gains access,
+as that would both be slow and allow Alice to connect his different personas.
```

Comment:
uc-minimalcredentials can be taken up independently of uc-whopermitted because uc-minimalcredentials' requirements can potentially disclose any specific set of agents, their class, or group. The example from uc-whopermitted should at most reveal what class of agents or groups can access - it strictly does not want to reveal individuals.

We can acknowledge uc-minimalcredentials, but needless to say, it doesn't entail that readable ACLs are required or even anything on the authorization layer. If this is truly about a shared understanding of what kind of credentials are acceptable, then it may be preferable to address this in a prior layer, and so not necessarily having to wire it up with WAC/ACLs. Consider the situation where a different Access Control mechanism is used. We'd still want to realise uc-minimalcredentials.